transitions/InvertedPageCurl.glsl

```diff
@@ -0,0 +1,214 @@
+// author: Hewlett-Packard
+// license: bsd 3-clause
```

Comment:
'bsd 3-clause' not found in supported licenses: MIT, EPL-1.0, CDDL-1.0, MPL-2.0, LGPL, GPLv2, GPLv3, Apache-2.0, BSD 2 Clause

Reply:
can it be "BSD 2 Clause" or do I need to add BSD 3? ^^

Reply:
i'm adding it anyway

Reply:
Well 3 clause is not the same as 2 clause ;-) I did not choose the license.

Reply:
ok I should have fixed it , can you rename it to exactly `BSD 3 Clause` ?

Reply:
I try